R/has-name.R

```diff
@@ -13,6 +13,5 @@
 #' has_name(iris, "Species")
 #' has_name(mtcars, "gears")
 #' @export
-has_name <- function(x, name) {
-  name %in% names2(x)
-}
+#' @name has_name
+has_name <- rlang::has_name
```

Comment:
We can either reexport, or implement a proper forwarder. The alias will be evaluated at installation time, which I don't think is ideal (even if it might not matter in this particular case).

Can you use `@inherit` to copy over documentation from `rlang`?